plugins/CoreVisualizations/JqplotDataGenerator/Chart.php

```diff
@@ -145,13 +150,14 @@ public function render()
         ProxyHttp::overrideCacheControlHeaders();
 
         // See http://www.jqplot.com/docs/files/jqPlotOptions-txt.html
-        $data = array(
-            'params' => array(
-                'axes'   => &$this->axes,
-                'series' => &$this->series
-            ),
-            'data'   => &$this->data
-        );
+        $data = [
+            'params' => [
+                'axes' => &$this->axes,
+                'series' => &$this->series,
+            ],
+            'data' => &$this->data,
```

Comment:
Maybe a comment that the data variable should have the same amount of items as archive states and they are 1-1

Reply:
Renamed `archiveStates` to `dataStates` to be clearer what is meant.